.github/workflows/infra-check.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           for f in $changed_files; do
             matched="no"
 
-            for infra_f in $INFRASTRUCTURE_FILES; do
+            for infra_f in "${INFRASTRUCTURE_FILES[@]}"; do
```

Comment:
Ahh I hate this hard to understand stuff on bash. Thanks!

Reply:
Oh, so do I - got it right after three tries.